v2/support/demo_get_set_app_metadata/assets/iframe.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="https://cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
+</head>
+<body>
+  <h3>Apps framework metadata test:</h3>
+  <label for="formName">Enter form name to save:</label>
+  <input style="width:250px" name="formName" id="formName" value="">
+  </br>
+  <input type="checkbox" id="myCheckbox">My checkbox</br>
+  <hr>
+  <button id=getMetaData>Get metadata</button>
+  </br>
+  <button id="setMetaData">Set metadata</button>
+  <p>NOTE: Remember to select 'Reload all apps' to retrieve newly saved values.</p>
+  <script src="https://static.zdassets.com/zendesk_app_framework_sdk/2.0/zaf_sdk.min.js"></script>
+  <script>
+  let client = ZAFClient.init()
+  client.invoke('resize', { width: '100%', height: '300px' })
+
+  // Don't know app's Installation ID yet.
+  let installationId = 0
+
+  function getMetaData() {
+    client.metadata().then((metadata) => {
+
+      console.log("metadata:", metadata)
+
+      // Need Installation ID when saving later.
+      console.log("installationId:", metadata["installationId"])
+      installationId = metadata["installationId"]
```

Comment:
This would only work if user clicks on "Get metadata" before clicking on "Set metadata", what if they click on "Set metadata" first?

Reply:
Added some logic to disable Set Metadata if Get hadn't been called first.